src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -659,7 +652,19 @@ def __init__(self, raw_response):
             totalCellCount=self.agg_contents(self.aggregates, 'total_cell_count', agg_form='value'),
             organTypes=list(map(OrganType.for_bucket, _organ_types['buckets'])),
             fileTypeSummaries=list(map(FileTypeSummary.for_bucket, _file_types['buckets'])),
-            cellCountSummaries=list(map(OrganCellCountSummary.for_bucket, _cell_counts['buckets'])))
+            cellCountSummaries=list(map(OrganCellCountSummary.for_bucket, _cell_counts['buckets']))
+        )
+
+    @classmethod
+    def _translate_aggregate(cls, aggregate: MutableMapping[str, MutableMapping]):
+        if 'field' in aggregate.get('meta', {}):
```

Comment:
This is always True at the moment. You know the `meta.field` entry is present for both call sites. And that is the bad smell. You need to refactor the to make this condition count. 

I would like this translation problem solved generically for any `terms` aggregation, not just specific ones, for individual fields. If someone adds another terms aggregation I don't want them to have to worry about the translation aspect.